templates/etc_named.conf.j2

```diff
@@ -38,6 +38,14 @@ options {
 
   rrset-order { order {{ bind_rrset_order }}; };
 
+{% if bind_zones | selectattr('rpz', 'defined') | selectattr('rpz', 'equalto', true) | list | length > 0 %}
+  response-policy {
+{% for zone in bind_zones | selectattr('rpz', 'defined') | selectattr('rpz', 'equalto', true) | list %}
+    zone "{{ zone.name }}";
+{% endfor %}
+  };
+{% endif %}
+
```

Comment:
As above, please change the variable name to `response policy`.

Reply:
Done